### ISSUE TITLE
NPE thrown when loading React 

### ISSUE THREAD

**Issue by EthanTG** [VolmitSoftware/React#402]
```
23.06 16:14:04 [Server] WARN java.lang.NullPointerException
23.06 16:14:04 [Server] WARN at com.volmit.react.controller.DataLogController.stop(DataLogController.java:48)
23.06 16:14:04 [Server] WARN at com.volmit.react.ReactPlugin.onDisable(ReactPlugin.java:336)
23.06 16:14:04 [Server] WARN at com.volmit.react.ReactPlugin$9.run(ReactPlugin.java:359)
23.06 16:14:04 [Server] WARN at com.volmit.react.util.TaskLater$1.run(TaskLater.java:37)
23.06 16:14:04 [Server] WARN at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftTask.run(CraftTask.java:71)
23.06 16:14:04 [Server] WARN at org.bukkit.craftbukkit.v1_12_R1.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:353)
23.06 16:14:04 [Server] WARN at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:739)
23.06 16:14:04 [Server] WARN at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406)
23.06 16:14:04 [Server] WARN at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679)
23.06 16:14:04 [Server] WARN at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577)
23.06 16:14:04 [Server] WARN at java.lang.Thread.run(Thread.java:745)
23.06 16:14:04 [Server] INFO Parsed 5t -> 250
23.06 16:14:04 [Server] INFO Loaded Goal: suppress-fluids.json (Suppress Fluids - Reduces fluid flows by React)
23.06 16:14:04 [Server] INFO Parsed 1m -> 60000
23.06 16:14:04 [Server] INFO Loaded Goal: purge-chunks.json (Purge Chunks - Purges excess chunks every minute. by React)
23.06 16:14:04 [Server] INFO Parsed 10t -> 500
23.06 16:14:04 [Server] INFO Loaded Goal: cull-entities.json (Cull Entities - Culls Entities in entity dense areas by React)
23.06 16:14:04 [Server] INFO Parsed 1s -> 1000
23.06 16:14:04 [Server] INFO Loaded Goal: suppress-redstone.json (Suppress Redstone - Reduces Redstone usage by React)
23.06 16:14:04 [Server] INFO React: Language: enUS
```

;(

**Comment by cyberpwnn** [comment]
Can be fixed temp with `/re reload -xcache`

**Comment by EthanTG** [comment]
the question is, can _you_ be fixed with that?

**Comment by cyberpwnn** [comment]
No, not with 1.8 servers around at least.

**Comment by EthanTG** [comment]
I see...

**Comment by cyberpwnn** [comment]
Fixed in 6.564
